src/lib/airtable/request.js

```diff
@@ -226,30 +226,27 @@ export const createManyInventorys = async (records) => {
 };
 
 // NONGENERATED: Create a Purchase Request and update the inventory's current qty (regardless of approval status)
-export const createPurchaseRequest = async (purchaseRequest) => {
+export const createPurchaseRequestAndUpdateInventory = async (purchaseRequest) => {
   let purchaseRequestId = "";
   const newQuantity = getInventoryCurrentQuantity(purchaseRequest.inventoryId) + purchaseRequest.amountPurchased;
   try {
-    const resp = await fetch(`${process.env.REACT_APP_AIRTABLE_ENDPOINT_URL}/purchase-request/create`, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(purchaseRequest)
-    })
-    console.log('Response for purchase request: ', resp);
-    await resp.json().then(data => purchaseRequestId = data.id);
+    delete purchaseRequest.id; // Remove the id field to add to Airtable
```

Comment:
Can you make a note that this doesn't handle offline yet? In particular I don't believe it handles the flow:
1. An inventory item was created while offline
2. While still offline, a purchase request was created for that inventory item (in this case the inventory item doesn't have an ID)

Let me know if i'm off the mark on that one. We can do that in a future PR